sdk/samples/iot/hub/src/paho_iot_hub_sas_telemetry_example.c

```diff
@@ -34,8 +34,15 @@
 #pragma warning(disable : 4996)
 #endif
 
+// Comment to use MQTT without WebSockets.
+#define USE_WEB_SOCKET
+#define TIMEOUT_MQTT_DISCONNECT_MS (10 * 1000)
+#define TELEMETRY_SEND_INTERVAL 1
+#define NUMBER_OF_MESSAGES 5
+#define SAS_TOKEN_EXPIRATION_TIME_DIGITS 4
+
 // DO NOT MODIFY: Device ID Environment Variable Name
-#define ENV_DEVICE_ID "AZ_IOT_DEVICE_ID"
+#define ENV_DEVICE_ID "AZ_IOT_DEVICE_ID_SAS"
```

Comment:
I think this should be left as is? Or is this to differentiate in an env so you can run both cert and sas devices in the same env.

Reply:
I'm unclear on the need for SAS here as well.

Reply:
Exactly for that reason: we need to be able to run all samples within a single command prompt. This isn't possible today because my x509 device is interfering with the one that does SAS.